torch/testing/_internal/common_utils.py

```diff
@@ -1414,8 +1417,53 @@ def f_retry(*args, **kwargs):
     return deco_retry
 
 
-# Methods for matrix generation
+# Methods for matrix and tensor generation
+
 # Used in test_autograd.py and test_torch.py
+def make_tensor(size, device: torch.device, dtype: torch.dtype, *,
+                low, high, requires_grad: bool = False) -> torch.Tensor:
+    """Returns a tensor of the specified size on the given device and dtype.
+       The tensors values are between -9 and 9, inclusive, for most dtypes,
+       unless low (high) is not None in which case the values are between
+       max(-9, low) and min(9, high).
```

Comment:
What if I pass in `low = 10` and `high = 20`? :D  Will that raise an error? Or give me a bunch of `9`?

Reply:
I'll add an assert.